runtime/pangolin/src/pallets/session.rs

```diff
@@ -25,16 +25,9 @@ sp_runtime::impl_opaque_keys! {
 	}
 }
 
-#[cfg(feature = "production")]
-frame_support::parameter_types! {
-	pub const Period: u32 = 6 * HOURS;
-	pub const Offset: u32 = 0;
-}
-#[cfg(not(feature = "production"))]
-frame_support::parameter_types! {
-	pub const Period: u32 = 2 * MINUTES;
-	pub const Offset: u32 = 0;
-}
+fast_runtime_or_not!(Period, ConstU32<{ 2 * MINUTES }>, ConstU32<{ 6 * HOURS }>);
+
+type Offset = ConstU32<0>;
```

Comment:
Nit pick: suggest using `ConstU32<0>` directly in the config below. Other changes LGTM.

Reply:
They are 2 places that use this.